src/FSLibrary/StellarKubeSpecs.fs

```diff
@@ -716,7 +721,7 @@ type NetworkCfg with
                 containers
 
         let containers =
-            if self.NeedNetworkDelayScript then
+            if self.missionContext.installNetworkDelay = Some true then
```

Comment:
no need to create the container if there's no location info (so might need to restore the function NeedNetworkDelayScript)

Reply:
Makes sense. I overwrote the commit and force-pushed it since this new commit almost reverts the original commit.